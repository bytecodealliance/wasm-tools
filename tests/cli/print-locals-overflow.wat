;; FAIL: print %

(module binary
    "\00asm" "\01\00\00\00"     ;; module header

    "\01"           ;; type section
    "\04"           ;; size of section
    "\01"           ;; one type
    "\60\00\00"     ;; function, no parameters or results

    "\03"   ;; function section
    "\02"   ;; size of function section
    "\01"   ;; one function
    "\00"   ;; type 0

    "\0a"   ;; code section
    "\09"   ;; size of code section
    "\01"   ;; 1 function
    "\07"   ;; size of function
    "\01"   ;; one local
    "\ff\ff\ff\ff\00"   ;; lots of this type
    "\70"   ;; type
)
