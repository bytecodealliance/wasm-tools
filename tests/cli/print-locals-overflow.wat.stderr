error: function exceeds the maximum number of locals that can be printed
